mailpoet/assets/js/src/segments/dynamic/templates/index.tsx

```diff
@@ -105,12 +79,17 @@ export function SegmentTemplates(): JSX.Element {
         </FlexItem>
       </Flex>
 
-      <TabPanel tabs={tabConfig}>
```

Comment:
Unrelated to this PR, but out of curiosity — a few lines above (can't put the comment there, unfortunately), there is the following snippet:

```
direction={['column', 'row'] as any} // eslint-disable-line @typescript-eslint/no-explicit-any -- typed as string but supports string[] and this is needed to make the component responsive
```

What does that mean? CSS flex doesn't seem to support anything like that, and the resulting HTML contains only the `row` value. Looking at the [original component source codes](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/flex/flex), I also can't find the support of such a setting 🤔

Reply:
I'm not sure exactly how this works internally, but passing `{['column', 'row'] as any}` means that `flex-direction` will be automatically updated to `column` when displaying the component on smaller screens. This is needed especially when the search box is visible otherwise one element is stacked on top of the other.

Reply:
@rodrigoprimo Oh, wow, interesting. Did you find that in some docs/code or somewhere?

Reply:
@JanJakes, I found it in the docs: https://wordpress.github.io/gutenberg/?path=/docs/components-flex--docs#responsive-direction